homeassistant/components/darksky/sensor.py

```diff
@@ -235,7 +248,11 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
             _LOGGER.warning("Monitored condition %s is deprecated", variable)
         if (not SENSOR_TYPES[variable][7] or
                 'currently' in SENSOR_TYPES[variable][7]):
-            sensors.append(DarkSkySensor(forecast_data, variable, name))
+            if 'alerts' not in variable:
```

Comment:
Put the alerts entity case first and the rest as else.

```py
if variable == 'alerts':
    ...
else:
    ...
```

Reply:
Fixed: 
```python
            if variable == 'alerts':
                sensors.append(DarkSkyAlertSensor(
                    forecast_data, variable, name))
            else:
                sensors.append(DarkSkySensor(forecast_data, variable, name))
```

Reply:
Fixed.  Thanks!
```python
            if variable == 'alerts':
                sensors.append(DarkSkyAlertSensor(
                    forecast_data, variable, name))
            else:
                sensors.append(DarkSkySensor(forecast_data, variable, name))
```